src/IssueViz/FixSuggestion/FixSuggestionHandler.cs

```diff
@@ -95,23 +95,28 @@ private void ApplySuggestedChanges(string absoluteFilePath, List<ChangesDto> cha
     {
         ideWindowService.BringToFront();
         var textView = documentNavigator.Open(absoluteFilePath);
+        var textEdit = textView.TextBuffer.CreateEdit();
         for (var i = changes.Count - 1; i >= 0; i--)
         {
             var changeDto = changes[i];
-            var textEdit = textView.TextBuffer.CreateEdit();
+            
             try
             {
                 var spanToUpdate = issueSpanCalculator.CalculateSpan(textView.TextSnapshot, changeDto.beforeLineRange.startLine, changeDto.beforeLineRange.endLine);
-                textView.Caret.MoveTo(spanToUpdate.Start);
-                textView.ViewScroller.EnsureSpanVisible(spanToUpdate, EnsureSpanVisibleOptions.AlwaysCenter);
+                if (i == 0)
+                {
+                    textView.Caret.MoveTo(spanToUpdate.Start);
```

Comment:
Consider extracting the lines 106-110 into a separate method. Clean code usually recommends to prevent having huge methods as this crowds the logic and makes it more difficult to understand.

Reply:
In this particular case, I think that it's better not to extract the if to a separate method. If I had to move something for clarity I would consider reworking the `try`. I would leave it as is at the moment and come back for refactoring when all the tasks affecting this piece are done.

Reply:
Will there be a coming back to the refactoring, though? There is no task for it...